psgeom/translate.py

```diff
@@ -704,18 +704,65 @@ def get_sign(v):
 ; however they are filled in with some decent defaults
 ; for the large CSPAD detector
 
-; clen =  /LCLS/detector0-EncoderValue
-; photon_energy = /LCLS/photon_energy_eV
-
-clen = 0.0
+clen =  /LCLS/detector_1/EncoderValue
+photon_energy = /LCLS/photon_energy_eV
 res = 9097.52
 adu_per_eV = 0.00338
 
-; data = /data/peakpowder
-
-; dim0 = %
-; dim1 = ss
-; dim2 = fs
+data = /entry_1/data_1/data
+
+; mask = /entry_1/data_1/mask
+; mask_good = 0x0000
+; mask_bad = 0xffff
+
+dim0 = %
+dim1 = ss
+dim2 = fs
+
+; The following lines define "rigid groups" which express the physical
+; construction of the detector.  This is used when refining the detector
+; geometry.
+
+rigid_group_q0 = q0a0,q0a1,q0a2,q0a3,q0a4,q0a5,q0a6,q0a7,q0a8,q0a9,q0a10,q0a11,q0a12,q0a13,q0a14,q0a15
```

Comment:
According to the docstring, it's designed to output cspad geometry only.
But I agree it would be good to generalize this to rayonix. 
If there was a detector.type we could compare, then the user wouldn't need to specify a flag.